src/command.c

```diff
@@ -360,9 +360,9 @@ bool cmd_frequency(target_s *t, int argc, const char **argv)
 	}
 	const uint32_t freq = platform_max_frequency_get();
 	if (freq == FREQ_FIXED)
-		gdb_outf("SWJ freq fixed\n");
+		gdb_outf("Debug interface frequency is not configurable.\n");
```

Comment:
Given this does get built into the firmware, perhaps `"Debug iface frequency is fixed\n"`? Just to try and conserve a few characters - we like the change otherwise and perhaps it might be worth `#if`'ing this and a shorter string on `PC_HOSTED == 1`.